.nvmrc

```diff
@@ -0,0 +1 @@
+22.14.0
```

Comment:
Side note: is there a way we could automate this being in sync w/ `package.json`?

Reply:
I think renovate will bump the version in `.nvmrc` and the `volta` field in `package.json` - https://docs.renovatebot.com/node/